client.go

```diff
@@ -33,7 +33,10 @@ func NewClient(ifi *net.Interface) (*Client, error) {
 	if err != nil {
 		return nil, err
 	}
+	return NewClientPacketConn(ifi, p)
+}
 
+func NewClientPacketConn(ifi *net.Interface, p net.PacketConn) (*Client, error) {
```

Comment:
Oh, I thought because you were reviewing PRs, you were a contributor. Oh well. At most I only wasted a small amount of time -- I probably would have held off until I was sure this was going to be accepted.